src/SDKs/Storage/Management.Storage/Microsoft.Azure.Management.Storage.csproj

```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Storage</PackageId>
 		<Description>Microsoft Azure Management Storage Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Storage</AssemblyName>
-		<Version>9.0.0-preview</Version>
+		<Version>9.1.0-preview</Version>
```

Comment:
@blueww When are you planning to release a stable version? Ideally you should have started preview versioning from `8.8.0-preview` and landed on `9.0.0` stable version. Do let us know when you are planning to move to version 10 so we can provide a better idea.
Not blocking on this